ninja-core/src/test/java/conf/Module.java

```diff
@@ -18,13 +18,26 @@
 
 import com.google.inject.AbstractModule;
 
+import ninja.utils.NinjaProperties;
+
 
 // Just a dummy for testing.
 // Allows to check that custom Ninja module in user's conf directory
 // works properly.
 public class Module extends AbstractModule {
 
-  
+	NinjaProperties ninjaProperties;
+	
+	public Module(NinjaProperties ninjaProperties) {
+		
+		// Will fail any test based on this Module if properties not received
+		if(ninjaProperties == null) {
+			throw new IllegalArgumentException("null properties");
```

Comment:
Guava precondition or a better error message? I did choose the second one ; pay attention it's the Module defined for the unit tests only.